source/index-management.rst

```diff
@@ -669,6 +669,13 @@ Common API Components
      * filter expression are included in the index.
      */
     partialFilterExpression: Document;
+
+    /**
+     * Optionally specifies a collation to use for the index. If not specified, the default collation
```

Comment:
If a user creates a collection object with a default collation set, you don't think the user would expect an index created via that collection object would apply its default collation?